### PR TITLE
Updated related person current name prefix patch

### PR DESCRIPTION
**Pull request by mani7002**
Description
----

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
image
image

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/example_json/r4_examples_relatedperson.rb [hunk 1]
```diff
@@ -789,6 +789,11 @@ module Resources
         'op': 'replace',
         'path': '/name/0/family',
         'value': 'Potter'
+      },
+      {
+        'path': '/name/0/prefix',
+        'op': 'replace',
+        'value': ['Mr.']
```
Comment (SantanuBanerjee) [inline lib/resources/example_json/r4_examples_relatedperson.rb:796]: Can the sequence be same as above, like,  op, path and then value?

Comment (Nandita92) [reply lib/resources/example_json/r4_examples_relatedperson.rb:796]: Updated here https://github.com/cerner/fhir.cerner.com/commit/1e0d55147568928481c491904b8ad3f917100689

Comment (Nandita92) [reply lib/resources/example_json/r4_examples_relatedperson.rb:816]: Updated here: https://github.com/cerner/fhir.cerner.com/commit/4de910767a99e9946ef8e2d341b8325dd0757d41

File: lib/resources/r4/related_person_patch.yaml [hunk 2]
```diff
@@ -159,3 +159,22 @@ operations:
...
+      {
+        "path": "/name/0/prefix",
+        "op": "replace",
+        "value": ["Mr."]
+      }
+    note: >
+      <ul>
+        <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `prefix` is intended to be replaced.</li>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+        <li>If no prefix value is provided the prefix on the name will be unset.</li>
```
Comment (Coffeevee) [inline lib/resources/r4/related_person_patch.yaml:179]: nit - 'If no prefix value is provided, the prefix on the name will be unset.'

File: lib/resources/r4/related_person_patch.yaml [hunk 3]
```diff
@@ -159,3 +159,22 @@ operations:
...
+    operation: replace
+    type: String
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#HumanName.prefix
+    description: Replace the `HumanName.prefix` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "path": "/name/0/prefix",
+        "op": "replace",
+        "value": ["Mr."]
+      }
```
Comment (exopsRamba) [inline lib/resources/r4/related_person_patch.yaml:174]: nit: Reorder like we did in the other example.

File: lib/resources/r4/related_person_patch.yaml (hunk 2 above)
Comment (Nandita92) [reply lib/resources/r4/related_person_patch.yaml]: Updated here: https://github.com/cerner/fhir.cerner.com/commit/5fa86f1cb8cec463ba0daa0d664bb5e112036137

File: lib/resources/r4/related_person_patch.yaml (hunk 3 above)
Comment (Nandita92) [reply lib/resources/r4/related_person_patch.yaml:174]: Updated here: https://github.com/cerner/fhir.cerner.com/commit/5fa86f1cb8cec463ba0daa0d664bb5e112036137

**Review by Coffeevee — APPROVED** [review]

**Review by sujanb12 — APPROVED** [review]

**Review by SantanuBanerjee — APPROVED** [review]

**Review by exopsRamba — APPROVED** [review]

File: lib/resources/r4/related_person_patch.yaml [hunk 4]
```diff
@@ -159,3 +159,23 @@ operations:
...
+        "op": "replace",
+        "path": "/name/0/prefix",
+        "value": ["Mr."]
+      }
+    note: >
+      <ul>
+        <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `prefix` is intended to be replaced.</li>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+        <li>If no prefix value is provided, the prefix on the name will be unset.</li>
+        <li>Only one prefix value should be provided.</li>
```
Comment (exopsRamba) [inline lib/resources/r4/related_person_patch.yaml:180]: Let's make this 'Only one prefix value may be provided.'

Comment (ShadabQuaiser) [reply lib/resources/r4/related_person_patch.yaml]: done https://github.com/cerner/fhir.cerner.com/commit/0e13ad2b36f313a1c2b42c087dfa0321cd2cec72

**Review by MadhurRaj — APPROVED** [review]

**Comment by MaxPhilips** [comment]
live
Screen Shot 2021-03-05 at 12 45 00 PM

**Pull request merged**